webhook/mutation/pod_mutator.go

```diff
@@ -237,6 +311,11 @@ func (m *podMutator) Handle(ctx context.Context, req admission.Request) admissio
 	}
 	pod.Annotations[dtwebhook.AnnotationInjected] = "true"
 
+	workloadName, workloadKind, workloadErr := rootOwnerPod(ctx, m.metaClient, pod, req.Namespace)
```

Comment:
is there a reason you don't reuse `err` here?

Reply:
There is no reason, err can be reused.